go/src/socialapi/models/helpers.go

```diff
@@ -260,6 +260,7 @@ func CreateAccountInBothDbsWithNick(nick string) (*Account, error) {
 			Salt:           accHex,
 			Name:           nick,
 			Email:          accHex + "@koding.com",
+			SanitizedEmail: accHex + "@koding.com",
```

Comment:
What is the difference between `SanitizedEmail` and `Email`? Is `SanitizedEmail` used anywhere? (can't see its usage in this PR)